### ISSUE TITLE
Request a connector to [capture from | materialize to] [your favorite system]

### ISSUE THREAD

**Issue by kirkholmes** [estuary/connectors#1614]
### System Name

Big Table

### Type

Both

**Comment by dyaffe** [comment]
Hey @kirkholmes, thanks for the request!

We're actually just prioritizing the Big Table materilaization connector now.  I'm curious if you could share a bit more about your use case though.

**Comment by dyaffe** [comment]
That's helpful.  It sounds like the most important thing for you would be to capture from Big Table then vs push to it.

**Comment by kirkholmes** [comment]
Yes. All of our use cases (so far) are to capture the data and store it in
Snowflake for processing and analysis.
